apps/client/src/types/metadata.ts

```diff
@@ -0,0 +1,8 @@
+export interface Metadata {
+  authenticationFlowType: string;
+  cognitoEndpoint: string;
+  identityPoolId: string;
+  region: string;
+  userPoolId: string;
+  userPoolWebClientId: string;
+}
```

Comment:
This is perfectly fine, yet since your metaTags object in `helpers.ts` is an object literal, there is another way to create this type. You can derive the type from the data by adding const assertion to `metaTags`, such like: 

```
const metaTags = {
  authenticationFlowType: '',
  cognitoEndpoint: '',
  identityPoolId: '',
  region: '',
  userPoolId: '',
  userPoolWebClientId: '',
} as const;
```

Then you can use the object as the type using `typeof` and when you want to add a new meta tag, you only need to update the object itself.

Reply:
That's cool. Good to know 😃 
Looking at the `apps/client/src/helpers/meta-tags.ts`, maybe I don't even need this interface. TS knows the type is 
```
{
    authenticationFlowType: string;
    cognitoEndpoint: string;
    identityPoolId: string;
    region: string;
    userPoolId: string;
    userPoolWebClientId: string;
}
```